.changeset/pink-jobs-accept.md

```diff
@@ -0,0 +1,18 @@
+---
+'@apollo/server-integration-testsuite': minor
+'@apollo/server': minor
+---
+
+Updating the ApolloServer constructor to take in a stringifyResult function that will allow a consumer to pass in a function that formats the result of an http query.
+
+Usage:
+
+        const server = new ApolloServer({
+          typeDefs,
+          resolvers,
+          stringifyResult: (value: FormattedExecutionResult) => {
+            let result = JSON.stringify(value, null, 10000) // modify response
+            result = result.replace('world', 'stringifyResults works!'); // replace text with something custom
+            return result;
+          },
+        });
```

Comment:
I can't use the suggest feature here due to character escaping but if you could surround the example with code fences for markdown readers that would be nice.
>\```ts
code example
\```

Maybe for the example we just provide the most common transform people use for pretty print? Not sure the replacement is going to be a common use case.
```ts
return JSON.stringify(value, null, 2);
```